src/plone/restapi/services/locking/configure.zcml

```diff
@@ -11,6 +11,14 @@
       name="@lock"
       />
 
+  <plone:service
```

Comment:
@sneridagh how did the lock features made it in here?

Reply:
They are needed, otherwise Plone Site does not have lock support.

No clue why "Plone Site" content type does not implement `IContentish`, but that's another fight.